lib/validation/validators/maxValue.ts

```diff
@@ -0,0 +1,13 @@
+import { helpers } from '@vuelidate/validators'
+
+export function maxValue(value: unknown, max: number) {
+  if (!helpers.req(value)) {
+    return true
+  }
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70